docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_customize_build_pipeline.adoc

```diff
@@ -0,0 +1,99 @@
+. Commit your changes to the repository of the component.
+
+
+== Extending the build pipeline with your own tasks
+
+.Procedure
+
+. In each `.yaml` file in the `.tekton` directory, add a new task under `tasks`. 
+
++ 
+Example task:
++
+[source]
+--
+name: example-task
+      params:
+      - name: example-param
+        value: “Example”
+      runAfter:
+      - build-container #You can be more specific by choosing another task
+      taskRef:
+        bundle: <path to container image> 
+        name: example-task
+      when:
+      - input: $(params.skip-checks)    #This references the pipeline parameters
+        operator: in
+        values:
+        - "false"
+--
++
+NOTE: If you want to define a task directly in this file, rather than using `taskRef`, you can use `taskSpec`. Visit the documentation linked at the end of this guide for further explanation.
+
++
+. Commit your changes to the repository of the component.
+
+
+== Verification
+
+When you commit changes to these `.yaml` files in your repository, {ProductName} automatically triggers a new build. Wait for {ProductName} to complete the new build, then verify your changes have been made by following these steps:
+
+. Navigate to *Activity > Pipeline runs*.
+. Select the most recent build pipeline run. 
+. In the *Details* tab, confirm that there are new tasks that you added in the pipeline visualization.
+. In the *Logs* tab, confirm the following:
+.. Any new tasks are in the navigation bar. 
+.. If you changed a parameter's value, and that value gets printed, the new value is in the log.
+
+== Troubleshooting
+
+If you experience any issues with your customized pipeline, try the following solutions:
+
+* If you believe that your desired parameter values are not being passed into the pipeline, make sure that your assignment of that value doesn't get overwritten later in the `.yaml` file.
+
+* If your new task is not appearing in the pipeline run, ensure the following:
+** You added it to the correct place in the `.yaml` files, so that it has the path `.spec.params` or `.pipelineSpec.params`. 
+** You specified a valid `runAfter` field, and that the task in that field completed successfully.
+
+* For problems with both parameters and tasks, make sure you committed your changes to the `.tekton` directory in the repository that {ProductName} references for the component.
+
+* If your build pipeline can no longer successfully run, your best option is to simply rebuild the `.tekton` directory:
+** Delete the `.tekton` directory in the repository of hte component.
+** In the {ProductName} console, on the *Overview* tab for the relevant application, scroll down and select *Manage build pipelines*.
+** Select the three dots next to the relevant component and select *Roll back to default pipeline*.
+** Complete the steps for xref:how-to-guides/configuring-builds/proc_upgrade_build_pipeline.adoc[upgrading the build pipeline] of that component again.
+
+== Additional Resources
```

Comment:
So, when we link to external documentation (anything that's not Red Hat documentation), we want to be careful. 

We want users to reference our documentation. It's not "bad" to link to other product's and companies' docs, but we want to be cognizant of why we're doing it. 

I don't disagree with it here, so I'm happy to keep it. But since you know the content more than I, if there's a reference doc here that's a "nice to have" but nothing super important for users to reference, then we can probably remove it.